ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -83,6 +83,28 @@ module Ldap = struct
   let string_of_user x =
     Rpcmarshal.marshal user.Rpc.Types.ty x |> Jsonrpc.to_string
 
+  let sid_of_name name =
```

Comment:
A better name could by `sid_from_ldap`

Reply:
there are some function like `name_of_sid`, I like to keep consistency.

This function is in module `Ldap`, I suppose this indicate query by ldap already.